front-end/src/main.js

```diff
@@ -0,0 +1,87 @@
+import EventHub from './eventhub/EventHub.js';
+import RecipeService from './services/recipeservice.js';
+import RecipeList from './components/recipelist/recipelist.js';
+import MyRecipes from './components/myrecipes/myrecipes.js';
+import Profile from './components/profile/profile.js';
+import RecipeDetail from './components/recipedetail/recipedetail.js';
+
+const app = document.getElementById('app');
+const eventHub = new EventHub();
```

Comment:
You aren't even using event hub outside of this instantiation? Idk if we need it or if theres further plans to implement

Reply:
Will need it in the future